engine/src/flutter/lib/web_ui/dev/firefox_installer.dart

```diff
@@ -29,8 +29,8 @@ Future<BrowserInstallation> getOrInstallFirefox(
   String requestedVersion, {
   StringSink? infoLog,
 }) async {
-  // These tests are aimed to run only on the Linux containers in Cirrus.
-  // Therefore Firefox installation is implemented only for Linux now.
+  // These tests are aimed to run only on the Linux containers on LUCI.
```

Comment:
Should this also be Linux/MacOS?

Reply:
I'll just remove the comment, it's never been in sync with the line of code below which reads just as well.

Reply:
Sounds good, it doesn't add any value.